teps/0096-pipelines-v1.md

```diff
@@ -332,6 +332,19 @@ Each V1 CRD should be fully documented before being released, but release of a V
 unrelated documentation issues.
 Once we've achieved the goals of this TEP, we can release a v1 software version of Pipelines.
 
+### Migration plan
+The v1 CRD is expected to be released and served in v0.42.0 while v1 would be the stored version in v0.43.0.
+A v1 client is provided for integration tests to be used for the migration.
+
+The [v1 CRD release](https://github.com/tektoncd/pipeline/pull/5579) could be used as a preview for the migration, 
+for them to start preparing for the migration. This includes: 
+- v1beta1 example migrating to v1
+- features in v1beta1 that are going to be defaulted as beta in v1
```

Comment:
I think you can say that the initial release with v1 CRDs served can serve as a v1 preview, so we can start migrating the CLI and dashboard, and then v1 will become the official version in the next release (meaning we should prioritize supporting v1 versions in the CLI, dashboard, and any changes to triggers if needed during this month). Describing the code PR in this TEP is a bit too much implementation detail IMO.